src/bio/bio_device.c

```diff
@@ -438,19 +450,29 @@ fetch_pci_dev_info(struct nvme_ctrlr_t *w_ctrlr, const char *tr_addr)
 		return -DER_INVAL;
 	}
 
+	w_ctrlr->pci_cfg    = NULL;
+	w_ctrlr->pci_cfg_sz = 0;
+	if (fetch_pci_cfg) {
+		D_ALLOC(w_ctrlr->pci_cfg, NVME_PCI_CFG_SPC_MAX_LEN);
+		if (w_ctrlr->pci_cfg == NULL)
+			return -DER_NOMEM;
+		w_ctrlr->pci_cfg_sz = NVME_PCI_CFG_SPC_MAX_LEN;
```

Comment:
Looks this 'pci_cfg_sz' isn't useful?

Reply:
correct isn't used anymore and forgot to remove it, done